webapp/src/components/Controls/Controls.tsx

```diff
@@ -207,6 +227,13 @@ const Controls: React.FC<Props> = ({
               onChange={handleDatasetSplitChange}
             />
           </Box>
+          <Box margin={1}>
+            <SwitchToggler
+              label="Without PostProcessing"
+              enable={postprocessing.withoutPostprocessing}
+              onChange={handlePostProcessingToggle}
+            />
```

Comment:
Final touch! The `SwitchToggler` component ended up being so simple that I doubt it is useful. I think I would define the `Switch` directly here. What do you think?
```suggestion
            <FormControlLabel
              control={
                <Switch
                  checked={postprocessing.withoutPostprocessing ?? false}
                  onChange={(_, checked) => handlePostprocessingChange(checked)}
                />
              }
              label={"Without PostProcessing"}
            />
```

Reply:
yes purpose is to make it reusable  but its the only place now its used now